src/common/io.ts

```diff
@@ -134,6 +143,9 @@ export class ARIOReadable implements AoARIORead {
     } else {
       throw new InvalidContractConfigurationError();
     }
+    this.paymentProvider = new TurboArNSPaymentProvider({
+      paymentUrl: config?.paymentUrl,
```

Comment:
i'd argue we don't need to expose this configuration option as it's a hard coded integration with our payment service. could always expose it later. is there a place where we need to set the payment URL from the client?

Reply:
having it injectable is required for interacting with payment.ardrive.dev -> testnet, which we should use for testing